src/FileSystem/FilesFinder.php

```diff
@@ -117,7 +117,7 @@ private function addFilterWithExcludedPaths(Finder $finder): void
 
         $finder->filter(function (SplFileInfo $splFileInfo) use ($excludePaths): bool {
             $realPath = $splFileInfo->getRealPath();
-            if ($realPath === '') {
+            if ($realPath === false) {
```

Comment:
How about cast `(string)` to keep existing behavour:

```suggestion
            $realPath = (string) $splFileInfo->getRealPath();
            if ($realPath === '') {
```

ref https://3v4l.org/bTvhO

Note: CI failing is waiting for https://github.com/rectorphp/phpstan-rules/pull/25 to be merged first.

Reply:
Says can't accept suggestion on deleted lines, so I pushed a commit. Thanks.